src/main/scala/sectery/producers/Html.scala

```diff
@@ -52,16 +53,35 @@ object Html extends Producer:
       case _ =>
         ZIO.effectTotal(None)
 
-  def getDescription(body: String): Option[String] =
+  def getTitle(doc: Document): Option[String] =
 
     def nonEmpty(x: String): Option[String] =
```

Comment:
Extract this into a shared private method; drop the duplicate one in `getDescriptions`.